### PR TITLE
#3: Prepare release of EVSCL

### LINKED ISSUE #3: Prepare EVSCL for release on Luarocks
## Situation

To use it as a proper base library, EVSCL needs to be available through Luarocks.

```[tasklist]
### Tasks
- [x] Get a unique namespace (`exasol.evscl`)
- [x] Fix requirement trace
- [x] Remove RLS-specific code
```

# Acceptance Criteria

* EVSCL available through Luarocks

### PR DESCRIPTION
**Pull request by redcatbear**
Close #3.

### CODE REVIEW

File: doc/system_requirements.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+# System Requirement Specification Exasol Virtual Schema Common for Lua
+
+## Introduction
+
+Exasol Virtual Schema Common for Lua (short "EVSCL") is base library for [Virtual Schemas](https://docs.exasol.com/db/latest/database_concepts/virtual_schemas.htm) that connect to Exasol databases.
```
Comment (kaklakariada) [inline doc/system_requirements.md]:
```suggestion
Exasol Virtual Schema Common for Lua (short "EVSCL") is a base library for [Virtual Schemas](https://docs.exasol.com/db/latest/database_concepts/virtual_schemas.htm) that connect to Exasol databases. 
```

File: doc/system_requirements.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+Virtual Schemas are conceptually related to database views. The main difference compared to database views is that Virtual Schema sources can be almost any data source. To make this possible, you need a so-called "Virtual Schema adapter", an Exasol extension that contains the logic to translate between the source and the projection provided by the Virtual Schema.
+
+From an end-user's perspective a Virtual Schema looks and feels like an internal schema of Exasol &mdash; albeit one that can only be read.
+
+## About This Document
+
+### Target Audience
+
+The target audience are end-users, requirement engineers, software designers and quality assurance. See section ["Stakeholders"](#stakeholders) for more details.
```
Comment (kaklakariada) [inline doc/system_requirements.md]: requirement engineers, software designers and quality assurance are not listed in the "Stakeholder" section

File: doc/system_requirements.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+The main work when creating a new virtual schema is to map the structure and data types of the data source to virtual schema metadata. This happens by scanning the data source and then reporting table and column structure back. This section holds requirements that are common for scanning an Exasol data source. 
+
+#### Reading Table Metadata From a Schema
+`req~reading-table-metadata-from-a-schema~1`
+
+EVSCL reads the list of tables contained in the source database schema.
+
+Needs: dsn
+
+#### Reading Columns Metadata From a Table
```
Comment (kaklakariada) [inline doc/system_requirements.md]:
```suggestion
#### Reading Column Metadata From a Table
```

File: doc/system_requirements.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+`req~reading-column-metadata-from-a-table~1`
+
+EVSCL reads the list of columns and their attributes contained from a table in the source database.
```
Comment (kaklakariada) [inline doc/system_requirements.md]:
```suggestion
EVSCL reads the list of columns and their attributes from a table in the source database.
```

File: doc/system_requirements.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+#### Include Tables
+`req~include-tables~1`
+
+EVSCL offers an interface to specify the tables from the source database schema that should be included in the virtual schema. 
+
+Needs: dsn
+
+### Shared Adapter Properties of Virtual Schemas That Access an Exasol Database
+
+Adapter properties allow users to configure their virtual schemas. In this section we define properties that all virtual schemas share which access an Exasol data source.
```
Comment (kaklakariada) [inline doc/system_requirements.md]:
```suggestion
Adapter properties allow VS Owners to configure their virtual schemas. In this section we define properties that all virtual schemas share which access an Exasol data source. 
```

**Review by kaklakariada — CHANGES_REQUESTED** [review]

File: doc/system_requirements.md (hunk 2 above)
Comment (redcatbear) [reply doc/system_requirements.md]: Added.

**Review by kaklakariada — APPROVED** [review]

**Pull request merged**